### DISCUSSION TITLE
peer class not working in tailwindcss

### DISCUSSION THREAD

**Discussion by salach** [tailwindlabs/tailwindcss#5171] [Help]
Hello
I am trying to use peer class with @apply but I keep getting peer class does not exist error. I have updated my tailwind to the latest version and yet the issue remains

**Comment by simonswiss** [comment]
Hey!

This is because the `peer` class technically doesn't "exist" – it doesn't define any CSS rules, but is instead used as a "marker" (or reference) in the HTML.

The same goes for `group` classes, you cannot use them in an `@apply` declaration because these classes are not actually generated by Tailwind.

What's generated is the classes that listen to state changes on the HTML element which has that class applied to it!

You can still use something like `peer-hover:bg-red-500` in your `@apply` declaration, but the `peer` class must be applied in your HTML markup, not in your CSS' `@apply`!

Hope it helps 👍

**Comment by Sanfored** [comment]
Well I can't say we were really planning on it but over the last few weeks we've been having a ton of fun dumping new and exciting features into Tailwind and now feels like the right time to cut a release, so here it is — Tailwind CSS v2.2!

[firstcallonline](www.firstcallonline.us/)
